scripts/o3de/o3de/repo.py

```diff
@@ -199,6 +203,7 @@ def process_add_o3de_repo(file_name: str or pathlib.Path,
     with file_name.open('w') as f:
         try:
             time_now = datetime.now()
+            # %H is 24 hour
             time_str = time_now.strftime('%d/%m/%Y %H:%M')
```

Comment:
I think it would be better to use ISO8601 timestamp format here as it is doesn't have ambiguity with regards to the user culture.